src/v2/guide/single-file-components.md

```diff
@@ -52,20 +52,22 @@ Incluso si no le gusta la idea de utilizar Single-File Components, usted puede a
 
-- **Modern JavaScript with ES2015/16**: Lea la guía de Babel [Learn ES2015](https://babeljs.io/docs/learn-es2015/). Usted no tiene que memorizar cada característica ahora mismo, pero mantenga esta pagina como una referencia a la cual pueda regresar.
+- **JavaScript moderno con ES2015/16** (Modern JavaScript with ES2015/16): Lee la guía de Babel [Aprende ES2015](https://babeljs.io/docs/learn-es2015/). No es necesario memorizar cada característica, pero mantén esta página como una referencia a la cual regresar.
 
-Despues de que usted haya tomado un día para profundizar en estos recursos, le recomendamos revisar la plantilla [webpack](https://vuejs-templates.github.io/webpack). Sigue las instrucciones y debería tener un proyecto Vue con componentes `.vue`, ES2015, y hot-reloading en un instante.
-
-Para aprender más sobre Webpack, visite [su documentación oficial](https://webpack.js.org/configuration/) y [Webpack Academy](https://webpack.academy/p/the-core-concepts). En Webpack, cada archivo puede ser transformado por un "loader" antes de ser incluido en el the bundle, y Vue ofrece el plugin [vue-loader](https://vue-loader.vuejs.org) para traducir single file componentes (`.vue`).
+Luego de tomarte un día para profundizar en estos recursos, te recomendamos consultar [Vue CLI 3](https://cli.vuejs.org/). Sigue las instrucciones y deberías tener un proyecto Vue con componentes `.vue`, ES2015, y recarga en caliente en unos pocos minutos!
 
 ### Para Usuarios Avanzados
 
-Ya sea que usted prefiera Webpack o Browserify, hemos documentado plantillas para ambos, tanto para proyectos simples como más complejos. Recomendamos visitar [github.com/vuejs-templates](https://github.com/vuejs-templates), tomando la plantilla correcta para usted, entonces siga las instrucciones en el README para generar un nuevo proyecto con [vue-cli](https://github.com/vuejs/vue-cli).
+La Interface de Línea de Comandos (CLI) se encarga de la mayoría de las configuraciones de herramientas para ti, pero también permite una personalización detallada a través de tus propias opciones de configuración.
+
+En caso de que prefieras definir tu configuración de compilación desde cero, deberás configurar manualmente Webpack con [vue-loader](https://vue-loader.vuejs.org/). Para obtener más información sobre Webpack, consulta su [Documentación Oficial](https://webpack.js.org/configuration/) y la [Academia Webpack](https://webpack.academy/p/the-core-concepts).
+
+Ya sea que prefieras Webpack o Browserify, hemos documentado plantillas para ambos, ya sea para proyectos simples o más complejos. Te recomendamos visitar [github.com/vuejs-templates](https://github.com/vuejs-templates), eligiendo la plantilla correcta para ti, y seguir las instrucciones en el archivo README para generar un nuevo proyecto con [vue-cli](https://github.com/vuejs/vue-cli).
```

Comment:
Some suggestions:
```suggestion
Ya sea que usted prefiera Webpack o Browserify, hemos documentado plantillas para ambos, ya sea para proyectos simples o más complejos. Le recomendamos visitar [github.com/vuejs-templates](https://github.com/vuejs-templates), eligiendo la plantilla correcta para usted, y seguir las instrucciones en el archivo _README_ para generar un nuevo proyecto con [vue-cli](https://github.com/vuejs/vue-cli).
```

Reply:
Changed to formal treatment throughout the document.